Adding the COMMUNICATION.md to the Standard Base Documentation pattern (#589)
gruetter/innerSourceTest: patterns/2-structured/project-setup/base-documentation.md

![CONTRIBUTING.md](../../../assets/img/standard-base-documentation/CONTRIBUTING-for-contributors.png)

### COMMUNICATION.md

Create a separate `COMMUNICATION.md` document. Link this document to your README.md so comprehensive contact information can be provided and not take up the extra space in your README. This document should answer frequently
asked questions about communicating with your team that contributors need to know. The goal is to streamline communications so users and contributors reach out to the correct person through a single channel. This reduces unnecessary distractions for team members and organizes communications so they do not get lost.

These communications can be broken into incoming and outgoing communications. Sections in the COMMUNICATION.md include:

> Points of contact for incoming communication and how to contact those users. Some examples include:

Reviewer: The formatting here is a bit confusing because two consecutive lines end with a colon. Any chance this can be formatted differently?

Author: Sections in the COMMUNICATION.md points of contact for incoming communications and information about outgoing communications from the project's ownership team. Included below are some examples.

Points of contact for incoming communication and how to contact those users:

![CONTRIBUTING.md](../../../assets/img/standard-base-documentation/CONTRIBUTING-for-contributors.png)

### COMMUNICATION.md

Create a separate `COMMUNICATION.md` document. Link this document to your `README.md` so comprehensive contact information can be provided and not take up the extra space in your README. This document should answer frequently
asked questions about communicating with your team that contributors need to know. The goal is to streamline communications so users and contributors reach out to the correct person through a single channel. This reduces unnecessary distractions for team members and organizes communications so they do not get lost.

> Sections in the COMMUNICATION.md points of contact for incoming communications and information about outgoing communications from the project's ownership team. Included below are some examples.

Reviewer: ```suggestion
Sections in the `COMMUNICATION.md` points of contact for incoming communications and information about outgoing communications from the project's ownership team. Included below are some examples.
```

> Sections in the `COMMUNICATION.md` points of contact for incoming communications and information about outgoing communications from the project's ownership team. Included below are some examples.

Reviewer: @kschueths I still don't get this sentence:
> Sections in the `COMMUNICATION.md` points of contact for incoming communications and information about outgoing communications from the project's ownership team.

Is there a verb missing?

Author: I can't seem to get to the place to edit it. I added a verb "include"

Sections in the `COMMUNICATION.md` include points of contact for incoming communications and information about outgoing communications from the project's ownership team. Included below are some examples.

Reviewer: I will add the fix for you.

For next time:
Hit this button here in the comment thread. That will give you a suggestion field within the comment window where you can make your edits. Those can then be committed directly in the comment thread as well.

![Screenshot 2023-09-14 at 21 25 24](https://github.com/InnerSourceCommons/InnerSourcePatterns/assets/163029/553b72e2-2984-46bf-84a6-8a18cd9f1f28)

Reviewer: ```suggestion
Sections in the `COMMUNICATION.md` include points of contact for incoming communications and information about outgoing communications from the project's ownership team. See some examples below.
```

* Europace AG - See blog post [InnerSource: Adding base documentation](https://tech.europace.de/post/innersource-base-documentation/)
* Paypal Inc.
* Mercado Libre - create a documentation site that contains how to get started with InnerSource and also define the basic artifacts that a repository must have to be InnerSource (README, CONTRIBUTING, CODING_GUIDELINES, etc).
* Analog Devices Inc.

## Authors

* Isabel Drost-Fromm

> * Katie Schueths

Reviewer: ```suggestion

## Acknowledgments

* Katie Schueths - for adding the `COMMUNICATION.md` concept
```

There is another PR where the same pattern is being extended.
There we introduced the 'Acknowledgements' section, so that we can thank people more specifically for the sections that they contributed.

I adapted this part here, so that we can more easily merge this with the changes from the other PR later.
